bin/git-create-branch

```diff
@@ -1,22 +1,30 @@
 #!/usr/bin/env bash
 
-for param in $*
+while test $# != 0
 do
-	case $param in
+	case $1 in
 		-r|--remote)
-			if [ $# -eq 2 ]
+			if [[ -n $2 ]]
 			then
-				BRANCH=$2
-			else			
 				REMOTE=$2
-				BRANCH=$3
+				shift
+			else			
+				REMOTE=origin
+				shift
```

Comment:
If I am right, the last argument would be shifted twice. Maybe we could remove it?